modular_skyrat/modules/emote_panel/code/emote_panel.dm

```diff
@@ -0,0 +1,978 @@
+/mob/proc/manipulate_emotes()
+	if(!mind)
+		return
+	var/list/available_emotes = list()
+	var/list/all_emotes = list()
+
+	// code\modules\mob\emote.dm
+	var/list/mob_emotes = list(
+		/mob/proc/emote_flip,
+		/mob/proc/emote_spin
+	)
+	all_emotes += mob_emotes
+	if(istype(src, /mob))
```

Comment:
use `ismob()`

Reply:
Appropriate check applied, also moved the checks to make it easier to read